pkg/gatherers/clusterconfig/machine_autoscaler.go

```diff
@@ -11,20 +11,29 @@ import (
 	"k8s.io/client-go/dynamic"
 )
 
-// GatherMachineAutoscalers collects MachineAutoscalers definition
+// GatherMachineAutoscalers Collects MachineAutoscalers definition.
 //
-// The Kubernetes api:
+// ### API Reference
+// - https://github.com/openshift/cluster-autoscaler-operator/blob/master/pkg/apis/autoscaling/v1beta1/machineautoscaler_types.go
+// - https://docs.openshift.com/container-platform/4.7/rest_api/autoscale_apis/machineautoscaler-autoscaling-openshift-io-v1beta1.html#machineautoscaler-autoscaling-openshift-io-v1beta1
 //
-//	https://github.com/openshift/cluster-autoscaler-operator/blob/master/pkg/apis/autoscaling/v1beta1/machineautoscaler_types.go
+// ### Sample data
+// - docs/insights-archive-sample/config/machineautoscalers/openshift-machine-api/worker-us-east-1a.json
 //
-// Response see:
+// ### Location in archive
+// - `config/machineautoscalers/{namespace}/{resource}.json`
 //
-//	https://docs.openshift.com/container-platform/4.7/rest_api/autoscale_apis/machineautoscaler-autoscaling-openshift-io-v1beta1.html#machineautoscaler-autoscaling-openshift-io-v1beta1
+// ### Config ID
+// `clusterconfig/machine_autoscalers`
 //
-// * Location in archive: config/machineautoscalers/{namespace}/{machineautoscaler-name}.json
-// * Id in config: clusterconfig/machine_autoscalers
-// * Since versions:
-//   - 4.8+
+// ### Released version
+// - 4.8.2
```

Comment:
This should be probably `4.8.0`

Reply:
So, I had the same question here. But it seems that the [BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1966523) and the [RN](https://access.redhat.com/errata/RHSA-2021:2438) points to `4.8.2`.